src/test/java/com/ui_init_setup/practiceproject/pages/ParaBankLoginComponent.java

```diff
@@ -0,0 +1,58 @@
+package com.ui_init_setup.practiceproject.pages;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.LoadableComponent;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class ParaBankLoginComponent extends LoadableComponent<ParaBankLoginComponent> {
+
+    private final WebDriver driver;
+    private final PageActions paraBankLoginPageActions;
+    public static final String PARA_BANK_URL = "https://parabank.parasoft.com/parabank/index.htm";
```

Comment:
do you plan on using this variable outside of this class?

Reply:
no intentions to use it outside of this class